astropy/io/ascii/core.py

```diff
@@ -259,19 +271,13 @@ class DefaultSplitter(BaseSplitter):
           for col_val in col_vals:
                ...
 
-    :param delimiter: one-character string used to separate fields.
-    :param doublequote:  control how instances of *quotechar* in a field are quoted
-    :param escapechar: character to remove special meaning from following character
-    :param quotechar: one-character stringto quote fields containing special characters
-    :param quoting: control when quotes are recognised by the reader
-    :param skipinitialspace: ignore whitespace immediately following the delimiter
     """
-    delimiter = ' '
-    quotechar = '"'
-    doublequote = True
-    escapechar = None
-    quoting = csv.QUOTE_MINIMAL
-    skipinitialspace = True
+    delimiter = ' '  #: one-character string used to separate fields.
+    quotechar = '"'  #: control how instances of *quotechar* in a field are quoted
+    doublequote = True  #: character to remove special meaning from following character
+    escapechar = None  #: one-character stringto quote fields containing special characters
+    quoting = csv.QUOTE_MINIMAL  #: control when quotes are recognised by the reader
+    skipinitialspace = True  #: ignore whitespace immediately following the delimiter
```

Comment:
My preference for attribute docstrings, which we've used in other places as well, is to put triple-quoted strings directly under the attribute.  For example:
```python
delimiter = ' '
"""one-character string used to separate fields."""
```

but it's up to @taldcroft to make the final call on that.  I'm more worried about why they aren't displaying properly in the attribute autosummary.  That should have been fixed by astropy/astropy-helpers#131